src/config/mod.rs

```diff
@@ -32,8 +35,10 @@ pub struct Config<'c> {
     metrics_path: Option<Cow<'c, Path>>,
     metrics_fifo: Option<Cow<'c, Path>>,
     pub(crate) jailer_workspace_dir: Cow<'c, Path>,
+    pub(crate) kernel_image_file: Cow<'c, Path>,
```

Comment:
`kernel_image_filename` would be better and consistent with `std` and `tokio` naming scheme.